kicks/index.html

```diff
@@ -116,26 +116,32 @@ <h1>LOOΠΔVERSE</h1>
     <div><iframe src="https://www.facebook.com/plugins/video.php?href=https%3A%2F%2Fwww.facebook.com%2Floonatheworld%2Fvideos%2F456693861374811%2F&width=500&show_text=false&appId=104171832569&height=375" width="500" height="375" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowTransparency="true" allow="encrypted-media" allowFullScreen="true"></iframe></div>
     <div class="summary">
 	<b><u>Kick:</u> 07</b>
-    <br><b><u>Date:</u></b> April 24, 2017
+    <br><b><u>Date:</u></b> 27 July 2017
```

Comment:
- Date for LOONA Kick is 27 July 2017 [[1]](https://www.facebook.com/loonatheworld/videos/456693861374811/)
  - Unix timestamp is 1501124400 (Line 44) <-> 27 July 2017 12:00PM KST